wallet-new/src/Cardano/Wallet/Kernel/DB/Spec/Pending.hs

```diff
@@ -197,6 +200,18 @@ liftMap2 :: (UnderlyingMap -> UnderlyingMap -> UnderlyingMap)
          -> Pending -> Pending -> Pending
 liftMap2 f (toMap -> p) (toMap -> p') = fromMap (f p p')
 
+{--------------------------------------------------------------------------------
+  Compression
+-------------------------------------------------------------------------------}
+
+type PendingDiff = InDb (Util.MapDiff Core.TxId Core.TxAux)
```

Comment:
I think this is much better now; I'd prefer for `PendingDiff` to be a newtype though.